migrations/versions/2022_050411_5269e8c2c2c9_.py

```diff
@@ -0,0 +1,86 @@
+"""Create partner tables and add the partner_id column to users
```

Comment:
Do you know if this file is generated using alembic?

Reply:
This is the migration that was written by hand (writing raw SQL) and then moved to alembic